.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+#### What does this PR do?
+
+[GPII-XXXX](https://issues.gpii.net/browse/GPII-XXXX)
+
+##### Description of the fix implemented
+
+##### Does this PR affect to the living environments? if so, how?
```

Comment:
Living environments?

Reply:
stg & prd

Reply:
I find this a bit redundant as it's currently phrased - the ticket will affect long-lived environments in a same way that is described in the previous sections. Perhaps smth. like _"Is any service disruption expected for staging and production?"_